cmd/vic-machine/configure/configure.go

```diff
@@ -217,6 +238,48 @@ func updateSessionEnv(sess *executor.SessionConfig, envName, envValue string) {
 	sess.Cmd.Env = newEnvs
 }
 
+func (c *Configure) processCertificates(networks map[string]*executor.NetworkEndpoint) error {
+
+	if !c.certificates.NoTLSverify && (c.certificates.Skey == "" || c.certificates.Scert == "") {
+		log.Infof("No certificate regeneration requested. No new certificates provided. Certificates left unchanged.")
+		return nil
+	}
+
+	if c.certificates.CertPath == "" {
+		c.certificates.CertPath = c.DisplayName
+	}
+
+	_, err := os.Lstat(c.certificates.CertPath)
+	if err == nil || os.IsExist(err) {
+		return fmt.Errorf("Specified or default certificate output location \"%s\" already exists. Specify a location that does not yet exist with --cert-path to continue or do not specify --tls-noverify if, instead, you want to load certificates from %s", c.certificates.CertPath, c.certificates.CertPath)
+	}
+
+	var debug int
+	if c.Debug.Debug == nil {
+		debug = 0
+	} else {
+		debug = *c.Debug.Debug
+	}
+
+	c.certificates.Networks = common.Networks{
+		ClientNetworkName:     networks["client"].Name,
+		ClientNetworkIP:       networks["client"].Assigned.String(),
```

Comment:
Here you should use the value from data object, instead of configuration. The assigned value is got from dhcp, which can only be set after VCH is started.
The certificate generation should use static ip set by user, which is in Data object, defined in Create struct.
You do not have that object defined in Configure struct, but that will be set after we read data value from configuration. In that process, we only care about the static ip as well.